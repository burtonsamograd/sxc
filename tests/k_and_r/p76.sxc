/* a reverse polish calculator slighly modified to be in one file */
(#include "./tests/k_and_r/p76.part1.sxc")
(#include "./tests/k_and_r/p76.part2.sxc")
(#include "./tests/k_and_r/p76.part3.sxc")
(#include "./tests/k_and_r/p76.part4.sxc")
     

	 
     
     