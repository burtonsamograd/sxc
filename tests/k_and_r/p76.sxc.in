1 1 +
2 2 +
2 2 -
10 1 /
10 2 /
10 10 *
