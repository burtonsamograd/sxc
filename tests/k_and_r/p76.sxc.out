	2
	4
	0
	10
	5
	100